<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Periodic File Transfer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	color: white;
	background-color:rgb(25, 35, 45);
    border: 1px solid rgb(69, 83, 100);
}

QLabel{
	border: 0;
	background-color: transparent;
}

QPushButton{
	background-color: rgb(69, 83, 100);
	border-radius: 4px;
    border: 0;
}

QPushButton:pressed{
	background-color: rgb(84, 104, 122);
	border-radius: 4px;
    border: 0;
}

QMainWindow &gt; QWidget &gt; QPushButton[text=&quot;Start&quot;]{
	background-color:rgb(104, 149, 102);
	min-width: 80px;
	max-width: 80px;
	min-height: 80px;
	max-height: 80px;
	border-radius: 40px;
}

QMainWindow &gt; QWidget &gt; QPushButton[text=&quot;Stop&quot;]{
	background-color: rgb(181, 74, 74);
	min-width: 80px;
	max-width: 80px;
	min-height: 80px;
	max-height: 80px;
	border-radius: 40px;
}

QTabWidget::pane{
    border: 0px;
}

QTabWidget::tab-bar{
	alignment: center;
}

QTabBar{
	border: 0;
}

QTabBar::tab {
    background-color: rgb(69, 83, 100);
    border: 0;
	border-radius: 4px;
	margin: 2px;
    min-width: 10ex;
	min-height: 5ex;
    padding: 2px;
}

QTabBar::tab:selected, QTabBar::tab:selected:hover{
    background:qlineargradient(spread:pad, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0.681818 rgba(84, 104, 122, 255), stop:0.7 rgba(52, 64, 216, 255));
	margin-bottom: 0;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 0px;
}

QTabBar::tab:hover {
	background-color: rgb(84, 104, 122);
}

QTabBar::tab:!selected {
    margin: 2px;
}

QScrollBar{
    border: 0;
    background: rgb(69, 83, 100);
    height: 3ex;
}

QScrollBar::handle {
    background: rgb(84, 104, 122);
    min-width: 20px;
}

QTableWidget {
    gridline-color: rgb(84, 104, 122);
	selection-background-color: rgb(84, 104, 122);
}

QHeaderView::section {
    background-color: rgb(84, 104, 122);
    border-style: none;
    border-bottom: 1px solid white;
}

QComboBox {
    border: 1px solid rgb(84, 104, 122);
    border-radius: 4px;
	background-color: rgb(84, 104, 122);
}

QComboBox:on {
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: margin;
    subcontrol-position: top right;
    min-width: 24px;
    border-left-width: 1px;
    border-left-color:rgb(69, 83, 100);
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow:on {
    top: 1px;
    left: 1px;
}

QComboBox::down-arrow {
	image: url(:/Icons/arrow-down-outline.svg);
}

QComboBox QAbstractItemView {
    border: 1px solid rgb(69, 83, 100);
    selection-background-color: rgb(84, 104, 122);
}

QLineEdit {
    selection-background-color: rgb(84, 104, 122);
}


QWidget:disabled{
    color: rgb(85, 104, 124);
}

QPushButton:disabled{
	background-color: rgb(34, 41, 49);
}

QLineEdit:disabled{
    background-color: rgb(34, 41, 49);
}

QComboBox:disabled{
    background-color: rgb(34, 41, 49);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bt_start_stop">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>855</width>
        <height>550</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="config_tab">
       <attribute name="title">
        <string>Configurations</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>213</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="lb_file_regex">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Expression to be matched:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_file_regex">
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="lb_period">
              <property name="text">
               <string>Period:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_period">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cb_period_unit">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Seconds</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Minutes</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Hours</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
            <property name="verticalSpacing">
             <number>3</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="lb_source_dir">
              <property name="text">
               <string>path/to/source/folder</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="bt_set_src_dir">
              <property name="text">
               <string>Select Source Folder</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="lb_destination_dir">
              <property name="text">
               <string>path/to/destination/folder</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QPushButton" name="bt_set_dest_dir">
              <property name="text">
               <string>Select Destination Folder</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="bt_swap_src_dst">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Swap</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>213</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="charts_tab">
       <attribute name="title">
        <string>Statistics</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </widget>
      <widget class="QWidget" name="logging_tab">
       <attribute name="title">
        <string>Logs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="styleresources.qrc"/>
 </resources>
 <connections/>
</ui>
